pkg/utils/addresses.go

```diff
@@ -213,8 +220,18 @@ func addressesDefaultInternal(af AddressFilter, getAddrs addressMapFunc, getRout
 			}
```

Comment:
Seems that code on line 218 should be updated

Reply:
Ah, yep. We're not populating matches in the loop anymore so that logic doesn't work.